examples/basic.py

```diff
@@ -27,7 +27,7 @@
 
 for key in env['featuresMap']:
     params = {"featureKey": key}
-    resp = s.post(url = 'http://localhost:8080/v1/activate', params=params, json=payload)
+    resp = s.post(url = 'http://localhost:8080/v1/isFeatureEnabled', params=params, json=payload)
```

Comment:
This is not an error. /v1/isFeatureEnabled does not exist and everything is routed through `/activate` for both features and experiments.